registry/storage/driver/azure/parser.go

```diff
@@ -45,5 +49,11 @@ func NewParameters(parameters map[string]interface{}) (*Parameters, error) {
 	if params.ServiceURL == "" {
 		params.ServiceURL = fmt.Sprintf("https://%s.blob.%s", params.AccountName, params.Realm)
 	}
+	if params.CopyStatusPollMaxRetry == 0 {
```

Comment:
question: what if the users don't wanna do retries and instead want to abort straight away

Reply:
I'm not sure why users would want that? But I guess if we want to support it then we could use -1 as the "no polling" config. WDYT?

Reply:
done